test-d/foundry/common/utils/helpers.mjs.test-d.ts

```diff
@@ -202,6 +202,44 @@ declare class ClassWithConstructorParameters {
 expectType<boolean>(foundry.utils.isSubclass(ClassWithNoConstructorParameters, ClassWithConstructorParameters));
 expectType<boolean>(foundry.utils.isSubclass(ClassWithConstructorParameters, ClassWithNoConstructorParameters));
 
+// expandObject
+// hard to infer back
+expectType<{}>(foundry.utils.expandObject({}));
+expectType<{ test: number }>(foundry.utils.expandObject({ test: 1 }));
+expectType<{ test: number; deep: { value: number } }>(
+  foundry.utils.expandObject<{ test: number; deep: { value: number } }>({
```

Comment:
Honestly this test isn't really testing anything to do with `expandObject` itself as a _function_, it's testing that the first generic parameter is the return Ideally we wouldn't even make these types of generic parameters visible, it's just a way to access the parameters when getting the output. I would just remove it as this isn't a desired way to see it used. The inference will obviously be wrong but I can update this with an `ExpandObject` type.